nautobot/dcim/forms.py

```diff
@@ -786,6 +786,26 @@ class ManufacturerFilterForm(NautobotFilterForm):
     platforms = DynamicModelMultipleChoiceField(queryset=Platform.objects.all(), to_field_name="name", required=False)
 
 
+#
+# Platform
+#
+
+
+class PlatformBulkEditForm(NautobotBulkEditForm):
+    pk = forms.ModelMultipleChoiceField(queryset=Platform.objects.all(), widget=forms.MultipleHiddenInput())
+    description = forms.CharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
+    name = forms.CharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
+
+    manufacturer = DynamicModelChoiceField(queryset=Manufacturer.objects.all(), required=False)
+
+    network_driver = forms.CharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
+    napalm_driver = forms.CharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
+    napalm_args = forms.JSONField(required=False, widget=forms.Textarea(attrs={"rows": 4}))
```

Comment:
Can you confirm that leaving this blank correctly does *not* clear out existing napalm_args on the edited records?